crates/signs/src/bars.rs

```diff
@@ -269,6 +290,26 @@ fn update_visibility_distance(
     }
 }
 
+fn update_visibility_timer(
+    parents: Query<&BarChild, With<Active>>,
```

Comment:
I think these are not needed, we can iterate directly over all `bars`. Or is there a reason for the indirection?

Reply:
Ah, that would be me editing code too quickly at 1 AM.